### PR TITLE
Feature/164 new tip event

### PR DESCRIPTION
**Pull request by ahoseinian**
Listen to tips.newTip and send push event on litentry_test network.

### CODE REVIEW

File: src/chain/config.ts [hunk 1]
```diff
@@ -33,6 +33,18 @@ const litentryNetwork: ChainConfig = {
         }
       };
     }
+  }, {
+    pattern: "tips.newTip",
+    getPushData() {
+      return {
+        topic: "tips.newTip",
+        data: { deeplink: "litentry://api/litentry_test/tips" },
```
Comment (silva-fj) [inline src/chain/config.ts]: All these events will have the same title and body for all networks. This payload should be constructed in a way that can be reused for all networks, the network should come from the env variable (if we decide to keep that approach). Later we can come up with a better format for the links

Comment (ahoseinian) [reply src/chain/config.ts]: I have changed this on 188e76c please check if it's better now.

**Review by geastwood — APPROVED** [review]
LGTM

File: src/chain/pushEvents.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { Event } from '@polkadot/types/interfaces';
+import * as admin from 'firebase-admin';
+
+export type PushEvent = { pattern: string; getPushData: (event: Event) => admin.messaging.TopicMessage };
+
+export const getPushEvents = ({ networkKey }: { networkKey: string }) => {
+  return [treasuryProposedEvent(networkKey), tipsNewTipEvent(networkKey)];
+};
+
+function tipsNewTipEvent(networkKey: string) {
```
Comment (silva-fj) [inline src/chain/pushEvents.ts]: The event configs can be just plain objects, currently you can get the network from `process.env.CHAIN_NETWORK`

Comment (ahoseinian) [reply src/chain/pushEvents.ts]: 7d54354 
done ✅

**Pull request merged**